openaq_api/openaq_api/main.py

```diff
@@ -255,7 +254,6 @@ def favico():
 app.include_router(countries.router)
 app.include_router(manufacturers.router)
 app.include_router(measurements.router)
-app.include_router(owners.router)
```

Comment:
Can we add this back in, i think this got inadvertently removed from the branching issue.